calibr8/core.py

```diff
@@ -537,15 +572,72 @@ def loglikelihood(self, *, y, x, theta=None):
             observed measurements (dependent variable)
         x : scalar, array-like or TensorVariable
             assumed independent variable
+        name : str
+            Name for the likelihood variable in a PyMC model (tensor mode).
+            Previously this was `f'{replicate_id}.{dependent_key}'`.
```

Comment:
Für murefi ist es bestimmt wichtig, dass das Namensschema so bleibt, oder? Wird das irgendwo sichergestellt? Wollen wir das dort implementieren?

Reply:
Wir greifen auf die likelihood Variable nirgends zu. Die Änderungen hier kommen aus #5  damit man direkten Einfluss auf die Benennung nehmen kann die auch im model_to_graphviz auftaucht.

Reply:
Ob wir das in murefi schon benutzen weiß ich gerade nicht. Sollten wir aber.
Würde die deprecation dann mit calibr8 7 umsetzen.
(Die Änderungen aus diesem und den verwandten PRs sind rückwärtskompatibel!)